src/GeneralStateTestsFiller/stMemExpandingEIP150Calls/OOGinReturnFiller.yml

```diff
@@ -0,0 +1,145 @@
+OOGinReturn:
+
+  env:
+    currentCoinbase: 2adc25665018aa1fe0e6bc666dac8fc2697ff9ba
+    currentDifficulty: '0x20000'
+    currentGasLimit: 0x100000000
+    currentNumber: "1"
+    currentTimestamp: "1000"
+    previousHash: 5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6
+
+  _info:
+    comment: Ori Pomerantz qbzzt1@gmail.com
+
+
+  pre:
+
+    ccccccccccccccccccccccccccccccccccccccc1:
+      balance: '0x0ba1a9ce0ba1a9ce'
+      code:   | 
+        {
+            [0] 0xDEAD60A7
+            (return 0 0x100)
+        }
+      nonce: '0'
+      storage: {}
+
+
+
+    ccccccccccccccccccccccccccccccccccccccc2:
+      balance: '0x0ba1a9ce0ba1a9ce'
+      code:   | 
+        {
+            [0] 0xDEAD60A7
+            (revert 0 0x100)
+        }
+      nonce: '0'
+      storage: {}
+
+
+    cccccccccccccccccccccccccccccccccccccccc:
+      balance: '0x0ba1a9ce0ba1a9ce'
+      code:   | 
+        {
+          ; Variables are 0x20 bytes (= 256 bits) apart, except for
+          ; code buffers that get 0x100 (256 bytes)
+          (def 'callRet    0x100)
+          (def 'type       0x120)
+          (def 'gas2Use    0x140)
+          (def 'retVal     0x160)
+
+          ; Other constants
+          (def 'NOP 0)   ; No OPeration
+
+          ; Understand the input.
+          [type]       $4
+          [gas2Use]    $36
+
+          ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+
+          [0] 0x60A760A7
```

Comment:
what if there is no memory set before the call? you mentioned there was some strange buffer